http-transport-binding.md

```diff
@@ -0,0 +1,338 @@
+# HTTP Transport Binding for Cloud Events
+
+Editor: Clemens Vasters (clemensv@microsoft.com)
+Date: 2018-04-05
+Version: WD-01
+
+(c) Copyright Cloud Events Authors
+
+## Abstract
+
+The HTTP Transport Binding for Cloud Events defines how events are mapped to
+HTTP 1.1 request and response messages.
+
+## Status of this document
+
+This document is a working draft.
+
+## Table of Contents
+
+1. [Introduction](#1-introduction)
+- 1.1. [Conformance](#11-conformance)
+- 1.2. [Relation to HTTP](#12-relation-to-http)
+- 1.3. [Content Modes](#13-content-modes)
+- 1.4. [Event Formats](#14-event-formats)
+- 1.5. [Security](#15-security)
+2. [Use of Cloud Events Attributes](#2-use-of-cloud-events-attributes)
+- 2.1. [contentType Attribute](#21-contenttype-attribute)
+- 2.2. [data Attribute](#22-data-attribute)
+3. [HTTP Message Mapping](#3-http-message-mapping)
+- 3.2. [Binary Content Mode](#31-binary-content-mode)
+- 3.1. [Structured Content Mode](#32-structured-content-mode)
+4. [References](#4-references)
+
+## 1. Introduction
+
+[Cloud Events][CE] is a standardized and transport-neutral definition of the
+structure and metadata description of events. This specification defines how
+the elements defined in the Cloud Events specification are to be used in
+[HTTP 1.1][RFC7230] requests and response messages.
+
+### 1.1. Conformance
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC2119][RFC2119].
+
+### 1.2. Relation to HTTP
+
+This specification does not prescribe rules constraining the use or handling of
+specific [HTTP methods][RFC7231-Section-4], and it also does not constrain the
+[HTTP target resource][RFC7230-Section-5-1] that is used for transferring or
+soliciting events.
+
+Events can be transferred with all standard or application-defined HTTP request
+methods that support payload body transfers. Events can be also be transferred
+in HTTP responses and with all HTTP status codes that permit payload body
+transfers.
+
+All examples herein that show HTTP methods, HTTP target URIs, and HTTP status
+codes are non-normative illustrations.
+
+This specification also applies equivalently to HTTP/2 ([RFC7540][RFC7540]),
+which is compatible with HTTP 1.1 semantics.
+
+### 1.3. Content Modes
+
+The specification defines two content modes for transferring events:
```

Comment:
"The" or "This"?  I assume you mean "This http mapping doc", so then "This" would be appropriate. With "The" I wasn't sure if you were referring to this doc or the CloudEvents spec.

Reply:
I mean "this"

Reply:
handling.